app/components/button.tsx

```diff
@@ -1,8 +1,9 @@
-import React from "react";
-import { ReactNode } from "react";
+import React, { ReactNode }  from "react";
 
-const Button = (props: { children: ReactNode }) => {
-  return <button>{props.children}</button>;
+type ButtonProps = React.HTMLAttributes<HTMLButtonElement>
```

Comment:
I would like to add a property here for size, how would I do that?

Reply:
I have updated this and added the property. I won't add it to the props for the component until we decide how it is to be used. https://github.com/dustinsoos/bored-user/pull/1/commits/d2d45613073e885bccd42e25bbc5ba8db906a36d